grafana/deployment.yaml

```diff
@@ -138,7 +145,7 @@ data:
         isDefault: true
         jsonData:
           authenticationType: gce
-          defaultProject: {{ global.gcp_project }}
+          defaultProject: $GCP_PROJECT
```

Comment:
I'm trying to figure out why this change would work. Can you explain it to me?

Reply:
I just learned that grafana allows [environment variables](https://grafana.com/docs/grafana/latest/administration/provisioning/#using-environment-variables) in their datasource templates and it will fill them in when grafana loads the yaml. So as long as `GCP_PROJECT` is set as an environment variable in the grafana container it should just work.